seeds/synth_email_domains.csv

```diff
@@ -0,0 +1,1101 @@
+name,weight
```

Comment:
why delete this and next file?

Reply:
Domains is one of ours. Used it to create the email address - firs_name, last_name, domain.
Do you think we will ever need a standalone domain? Maybe yes. In this case we can keep it as a separate column.